### PR TITLE
Fixes #11796: System groups should display their list of nodes 

### PR DESCRIPTION
**Pull request by fanf**
https://issues.rudder.io/issues/11796

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala [hunk 1]
```diff
@@ -150,7 +167,23 @@ class NodeGroupForm(
...
+        showFormNodeGroup(group)
+    })(html)
+  }
+
+  private[this] def showRulesForTarget(target: SimpleTarget): NodeSeq = {
+    val noDisplay = DisplayColumn.Force(false)
+    val cmp = new RuleGrid("rules_for_current_group", None, false, None, noDisplay, noDisplay)
+    val rules = dependencyService.targetDependencies(target).map( _.rules.toSeq.filter(!_.isSystem)).toOption
+    cmp.rulesGridWithUpdatedInfo(rules, false, false)
+}
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala]: nitpick time: curly brace indentation is not ok

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala [hunk 2]
```diff
@@ -150,7 +167,23 @@ class NodeGroupForm(
...
+
+  private[this] def showFormNodeGroup(nodeGroup: NodeGroup): CssSel = {
+(
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala]: Parenthesis should be idented at the method level

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala [hunk 3]
```diff
@@ -254,45 +324,56 @@ class NodeGroupForm(
   }
 
   private[this] def onSubmit() : JsCmd = {
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala]: Maybe we can hide the submit button for RuleTargets ?

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNodeGroupTree.scala [hunk 4]
```diff
@@ -215,7 +215,7 @@ object DisplayNodeGroupTree extends Loggable {
         onClickNode match {
           case None                      => <a style="cursor:default">{xml}</a>
           // only disable click when actions are empty so we can act on them
-          case _ if(targetInfo.isSystem && targetActions.isEmpty) => <a style="cursor:default">{xml}</a>
+          //case _ if(targetInfo.isSystem && targetActions.isEmpty) => <a style="cursor:default">{xml}</a>
```
Comment (VinceMacBuche) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNodeGroupTree.scala]: Remove this line

**Comment by fanf** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala (hunk 3 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala]: It his hidden in the `showFormTarget` method, here it's just a security if for whatever reason the js is triggered. Given the code structure, I wasn't ablt to make that state unrepresentable.

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala (hunk 2 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala]: done

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala (hunk 1 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/NodeGroupForm.scala]: done

**Comment by fanf** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNodeGroupTree.scala (hunk 4 above)
Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNodeGroupTree.scala]: done

**Comment by fanf** [comment]
Commit modified

**Comment by fanf** [comment]
Commit modified

**Pull request merged**